flink-cdc-e2e-tests/flink-cdc-pipeline-e2e-tests/src/test/java/org/apache/flink/cdc/pipeline/tests/utils/PipelineTestEnvironment.java

```diff
@@ -114,30 +141,42 @@ public static List<String> getFlinkVersion() {
     public void before() throws Exception {
         LOG.info("Starting containers...");
         jobManagerConsumer = new ToStringConsumer();
-
-        String flinkProperties = getFlinkProperties();
-
+        // These cmds will put the FLINK_PROPERTIES to flink-conf.yaml
+        List<String> cmds = new ArrayList<>();
+        cmds.add("cp /opt/flink/conf/flink-conf.yaml /opt/flink/conf/flink-conf.yaml.tmp");
+        for (String prop : EXTERNAL_PROPS) {
+            cmds.add(String.format("echo '%s' >> /opt/flink/conf/flink-conf.yaml.tmp", prop));
+        }
+        cmds.add("mv /opt/flink/conf/flink-conf.yaml.tmp /opt/flink/conf/flink-conf.yaml");
```

Comment:
Should this be `config.yaml` as we're at Flink 1.19+?

Also, shouldn't these configurations take effect as we're setting `.withEnv("FLINK_PROPERTIES", FLINK_PROPERTIES)`? Why we need to modify the configuration YAML file manually